principles/fp-011-locus-of-authority.md

```diff
@@ -3,7 +3,7 @@ layout: principle
 id: fp-011-locus-of-authority
 title: Locus of Authority (principle 11)
 ---
-
+CONTENTS: [Recommendations/Requirements](https://obofoundry.org/principles/fp-011-locus-of-authority.html#Recommendations) | [Implementation](https://obofoundry.org/principles/fp-011-locus-of-authority#Implementation) | [Examples/Counter-Examples](https://obofoundry.org/principles/fp-011-locus-of-authority.html#Counter-Examples) | [Criteria for Review](https://obofoundry.org/principles/fp-011-locus-of-authority.html#Criteria) | [Feedback/Discussion](https://obofoundry.org/principles/fp-011-locus-of-authority.html#Feedback) 
```

Comment:
@erik-whiting before merging make sure you fix this to relative links!

Reply:
(Also wait for at least one EWG review before merging!)

Reply:
@matentzn I think it should be the PR author's responsibility to use the correct links